packages/vitest/package.json

```diff
@@ -123,6 +123,9 @@
     "safaridriver": {
       "optional": true
     },
+    "playwright": {
```

Comment:
Should also be specified in `peerDependencies` with webdriverio and safaridriver (we missed it last time)